tests/test_basic_ops_list.py

```diff
@@ -31,11 +31,24 @@ def test_list_of_dicts() -> None:
     assert c[1].key2 == "value2"
 
 
-def test_list_get_with_default() -> None:
-    c = OmegaConf.create([None, "???", "found"])
-    assert c.get(0, "default_value") == "default_value"
-    assert c.get(1, "default_value") == "default_value"
-    assert c.get(2, "default_value") == "found"
+@pytest.mark.parametrize("default", [None, 0, "default"])
+@pytest.mark.parametrize(
+    ("cfg", "key", "expected"),
+    [
+        (["found"], 0, "found"),
+        ([None], 0, None),
+        (["???"], 0, "__DEFAULT__"),
+    ],
+)
+def test_list_get_with_default(
+    cfg: List[Any], key: int, expected: Any, default: Any
+) -> None:
+    c = OmegaConf.create(cfg)
+    val = c.get(key, default_value=default)
+    if expected == "__DEFAULT__":
+        assert val == default
+    else:
+        assert val == expected
```

Comment:
I know I have done it myself before, but please split the new test into two tests.
One when the expected value is the provided default and another where the expect value is given by the test parameterization.

Reply:
Done in 9f06325 (also added a few more test cases at the same time)